src/cisco_gnmi/xe.py

```diff
@@ -318,7 +482,7 @@ def parse_xpath_to_gnmi_path(self, xpath, origin=None):
         """
         if origin is None:
             # naive but effective
-            if ":" not in xpath:
+            if "openconfig" in xpath:
```

Comment:
This does not account for desired `openconfig` origin pathing of `/interfaces/interface/...` - not module prefixed. Let's think about this.